internal/providers/google_chat/alerts.go

```diff
@@ -26,12 +27,8 @@ type AlertDetails struct {
 	UUID     uuid.UUID
 }
 
-// ChatMessage represents the structure for sending a
-// Text message in Google Chat Webhook endpoint.
-// https://developers.google.com/chat/api/guides/message-formats/basic
-type ChatMessage struct {
-	Text string `json:"text"`
-}
+// https://github.com/googleapis/google-api-go-client/blob/main/chat/v1/chat-gen.go
+type ChatMessage = chatv1.Message
```

Comment:
I have the impression that many Go developers will not understand this syntax, because type aliasing is not used a lot. I would suggest to consider adding a comment like

> Type alias, see https://go.dev/blog/alias-names

(feel free to disagree and discard this suggestion)

Reply:
Thank you for your review.
Added a comment.

I used aliases so that it wouldn't have to change the parts where `ChatMessage` is used.
However, if aliases are not used a lot, it might be better to change `ChatMessage` to `Message` since there aren't many of them.

How about changing `ChartMessage` to `Message`?

Reply:
@marco-m-pix4d Thank you for your comment.

I changed the `ChatMessage` to `Message`.

Reply:
Yep, I agree `Message` is better.